app/uk/gov/hmrc/thirdpartyorchestrator/domain/models/applications/AccessType.scala

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.thirdpartyorchestrator.domain.models.applications
+
+import uk.gov.hmrc.thirdpartyorchestrator.domain.utils.EnumJson
+
+object AccessType extends Enumeration {
```

Comment:
We need to get rid of enumertion and move to sealed trait with a companion object that contains the extending case classes/objects

Reply:
Removed